php-base/Dockerfile

```diff
@@ -92,6 +92,7 @@ RUN mkdir -p $APP_DIR $UPLOAD_DIR $SESSION_SAVE_PATH \
 # Put other config and shell files into place.
 COPY nginx.conf fastcgi_params gzip_params $NGINX_DIR/
 COPY nginx-app.conf $NGINX_USER_CONF_DIR
+RUN touch "${NGINX_USER_CONF_DIR}/nginx-http.conf"
```

Comment:
Not a blocker, but if we add an empty file and combine it with the above COPY, it's one fewer docker layer.